examples/polyglot-zipkin/java-dropwizard/README.adoc

```diff
@@ -0,0 +1,17 @@
+= Example application of Dropwizard with brave
+
+Dropwizard application instrumented with zipkin brave library.
+
+== Run & Build
+[source,shell]
```

Comment:
Seems like the primary way to run this whole example is via docker, so might be best to have the docker instructions as the first prominent instructions?

Then for those services that can also be run standalone, possibly make sure it is a clearly separate instruction - possibly names "Running service standalone"?